OpenEdXMobile/src/main/java/org/edx/mobile/module/analytics/ISegment.java

```diff
@@ -155,7 +153,6 @@ Properties trackVideoOrientation(String videoId, Double currentTime,
         String SUPPORTED = "supported";
         String NEW_OUTLINE_MODE = "new-mode";
```

Comment:
Yeah, looks like it was introduced in #332, and removed as redundant in #369 (but the keys were left out). I have removed the key in commit 2cc3a70f98c5244d6049f7622e6bcc8455244f26.